assets/script/script.js

```diff
@@ -4,33 +4,50 @@ const OPENWEATHER_API_KEY = "5b1c716e64155c6f31f83fc752ff2b1f";
 let weatherWord;
 let city;
 
-$(document).ready(function() {
-  // listen for city input
-  // until front-end is search input is hooked up, short-circuit with Minneapolis
-  $("#searchBtn").on("click", function() {
-    $("#weather-music-list").empty();
+$(document).ready(function () {
+  let srchHist = JSON.parse(window.localStorage.getItem("history")) || [];
+  for (let i = 0; i < srchHist.length; i++) {
+    displayHist(srchHist[i]);
+  }
+
+  $("#searchBtn").on("click", function () {
+    $("#display").empty();
     city = $("#citySearch").val();
-    // get weather word for current weather in that city
-    let weatherApiUrl = `https://api.openweathermap.org/data/2.5/weather?q=${city}&units=imperial&appid=${OPENWEATHER_API_KEY}`;
+    if (city !== "") {
```

Comment:
this can be simplified a little...
`if (city) {...`
it will return true if `city` has been set.